pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -60,7 +60,6 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
 	is := makeInjectionSet(binding, cred)
-
 	if err := b.injectSecret(is.secret); err != nil {
```

Comment:
@duglin changes made in https://github.com/kubernetes-incubator/service-catalog/pull/256/commits/c3ef033a96a5a06c34fba7d6276d15fb0d33900f to simplify this file. thanks for prompting me